src/Uno.Extensions.Localization/LocalizationService.cs

```diff
@@ -53,7 +53,7 @@ private void ApplyCurrentCulture(bool updateThreadCulture = true)
 				((CurrentCulture is not null) ?
 				PickSupportedCulture(CurrentCulture) :
 				PickSupportedCulture(CultureInfo.CurrentCulture)) ?? new CultureInfo(DefaultCulture);
-			ApplicationLanguages.PrimaryLanguageOverride = culture.TwoLetterISOLanguageName;
+			ApplicationLanguages.PrimaryLanguageOverride = culture.Name;
```

Comment:
It will always be `Name` @nickrandolph or in some other scenarios, will some other options like TwoLetterISOLanguageName be needed? 
Do you think it will be helpful to have a property on the LocalizationService that can help choose what's needed here at the end?

Reply:
The [documentation on PrimaryLanguageOverride](https://learn.microsoft.com/en-us/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride) states that it should be a [BCP-47](https://tools.ietf.org/html/bcp47) language tag. 
The remarks in the [documentation for CultureInfo](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.name?view=net-6.0#remarks) states that "Culture names follow the standard defined by BCP-47"